manifests/poc-data-cluster/prod/spark/iot-payload-totals-silver.yaml

```diff
@@ -0,0 +1,83 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: iot-payload-totals-silver-query
+  namespace: spark
+data:
+  query.sql: |
+    SELECT 
+      date, sum(payload_size) AS payload_total
+    FROM TABLE_IOT_PACKETS
```

Comment:
Have you tested these? I don't think you can do an aggregate in spark streaming without a window

Reply:
It does not need to be an aggregate over a 24-hour period.  The count(region) could be over whatever time period the job chooses.  However the job windows do need to be non-overlapping.  How can we guarantee non-overlapping windows?